agagd/agagd_core/admin.py

```diff
@@ -1,8 +1,17 @@
-from agagd_core.models import Chapters, Country, Game, Member, MembersRegions, Membership
+from agagd_core.models import (
+    Chapters,
+    Country,
+    Game,
+    Member,
+    MembersRegions,
+    Membership,
+)
 from django.contrib import admin
 
-class MemberAdmin(admin.ModelAdmin): 
-    list_display = ('member_id', 'full_name', 'join_date', 'chapter', 'chapter_id')
+
+class MemberAdmin(admin.ModelAdmin):
+    list_display = ("member_id", "full_name", "join_date", "chapter", "chapter_id")
```

Comment:
Any chance I could convince you to go single quotes?

Reply:
So this is based using black: https://github.com/psf/black/blob/d62d677ca2135714d6159f6b4839983fb0a557c8/docs/the_black_code_style.md#strings. I don't have a preference, but I think consistency here would be important. I don't feel it changes too much.

Reply:
Agree that we should go for consistency.

Looks like the way achieve the single quotes is `--skip-string-normalization`.  If I'm reading this right, looks like it might be the [`args`](https://pre-commit.com/#plugins) part of the `black` hook?

Reply:
Yeah.

> This is meant as an adoption helper, avoid using this for new projects.

I think it could be add as args to .pre-commit-hooks.yaml. (This is meant as an adoption helper, avoid using this for new projects.)